orchestrator/services/processes.py

```diff
@@ -515,8 +551,17 @@ def thread_resume_process(
         user_inputs = [{}]
 
     pstat = load_process(process)
-    if not pstat.workflow.authorize_callback(user_model):
-        raise_status(HTTPStatus.FORBIDDEN, error_message_unauthorized(str(process.workflow_name)))
+
+    auth_resume, auth_retry = get_auth_callbacks(pstat)
+    #TODO if retrying a WAITING step, can I detect that the system is retrying automatically?
+    # Possibly user=None means no auth and retry button, user=SYSTEM means resume_all.
+    # What about retry when logged in?
```

Comment:
This has proven tricky to experiment with. The example orchestrator is currently non-operational, and I've had trouble loading these changes into our own orchestrator instance. Guidance would be appreciated on how I can distinguish a workflow being retried manually vs. resumed from WAITING.

Reply:
I was able to resolve this by lifting this check up to `resume_process_endpoint`. This "fixes" the issue by restricting the scope of what's being considered. There's still the matter of adding RBAC to `resume_all_processess_endpoint`, but that might be a bit trickier to do and needs specification.

I'll make sure that's addressed in a ticket somewhere.